index.html

```diff
@@ -350,8 +350,9 @@ <h2>`as` attribute</h2>
       </div>
     </section>
     <section>
-      <h2>Server Push (HTTP/2)</h2>
-      <p>HTTP/2 ([[!RFC7540]]) allows a server to pre-emptively send ("push")
+      <h2>Server Push (HTTP/2 and HTTP/3)</h2>
```

Comment:
Better to remove the "HTTP/2" from the title

Reply:
Updated